simple_game/simple_game.ino

```diff
@@ -207,7 +209,10 @@ void goalReached(int x, int y) {
     
     double time = t / 1000.;
     drawGameOverScreen(time);
+    
     /* add submit score code here */
+    Serial.println("You have reached the gole.");
```

Comment:
*spelling: gole -> goal

Reply:
thx haha

Reply:
well  I fixed it in my local repository.
And I think because it's just a statement used for test and will not be output to the screen,
we don't have to bother making a new pull request.